pages/user/[username]/card.tsx

```diff
@@ -20,22 +26,107 @@ export const getServerSideProps: GetServerSideProps<CardProps> = async (context)
   };
 };
 
-const STUB_DATA = {
-  username: "foxyblocks",
-  name: "Chris Schlensker",
-  avatarURL: "https://avatars.githubusercontent.com/u/555044?v=4",
-  prs: 2,
-  contributions: 57,
-  bio: "This is the story all about how my life got flipped turned upside down, and I'd like to take a minute just sit right there, I'll tell you how I became the prince of a town called Bel-Air.",
-  activity: "high",
-  prVelocity: 102,
-} as const;
+const cardData = [...STUB_DEV_CARDS];
+
+const springFunction = (selectedIndex: number) => (index: number) => ({
+  opacity: index === selectedIndex ? 1 : 0,
+  transform: `translate3d(${index === selectedIndex ? 0 : 100}%, 0, 0)`,
+  config: {
+    easing: easings.easeInOutElastic,
+  },
+});
 
 const Card: NextPage<CardProps> = ({ username }) => {
+  const [selectedUserName, setSelectedUserName] = useState<string>(username as any);
+  const iframeTransition = useTransition(selectedUserName, {
+    from: { opacity: 0, transform: "translate3d(100%, 0, 0)" },
+    enter: { opacity: 1, transform: "translate3d(0, 0, 0)" },
+    leave: { opacity: 0, transform: "translate3d(100%, 0, 0)" },
+  });
+
   return (
-    <div>
-      <h1>Stub Card Page</h1>
-      <DevCard isLoading={false} {...STUB_DATA} />
+    <div
+      style={{
+        background: `url(${BubbleBG.src}) no-repeat center center, linear-gradient(147deg, #212121 13.41%, #2E2E2E 86.8%)`,
+        backgroundSize: "cover",
+      }}
+    >
+      <main className="w-full h-full min-h-screen grid md:pb-20" style={{ gridTemplateRows: "auto 1fr auto" }}>
+        <div className="grid justify-center place-content-start py-7 px-3 md:justify-start">
+          <HeaderLogo withBg={false} />
+        </div>
+        {/* center content, two columns on desktop */}
+        <div
+          className="grid place-content-center"
+          style={{
+            gridTemplateColumns: "34% auto",
+            columnGap: "4%",
+          }}
+        >
+          <div className="flex items-center md:justify-end">
+            <div className="flex flex-col gap-10">
+              <DevCardCarousel isLoading={false} cards={cardData} onSelect={(name) => setSelectedUserName(name)} />
+              <div className="hidden md:flex align-self-stretch justify-center">
+                <Button variant="primary" className="justify-center">
+                  Create your own dev card!
+                </Button>
+              </div>
+            </div>
+          </div>
+          <div className="hidden md:grid overflow-hidden">
+            {iframeTransition((style, item) => {
```

Comment:
will changing `item`  to `username` for readability reasons break anything?
```suggestion
            {iframeTransition((style, username) => {
```